Example/PaymentSheet Example/PaymentSheet Example/PlaygroundController.swift

```diff
@@ -107,7 +107,6 @@ class PlaygroundController: ObservableObject {
         if settings.userOverrideCountry != .off {
             configuration.userOverrideCountry = settings.userOverrideCountry.rawValue
         }
```

Comment:
Seems like we should update the controller to turn on/off this feature?

Reply:
Will do this in another PR that adds UI tests.  Shame to take up real estate for such a small and _almost_ unused property though, our test playground settings UI isn't scaling so well these days.